packages/eslint-plugin/__tests__/stylex-valid-styles-test.js

```diff
@@ -583,6 +598,72 @@ eslintTester.run('stylex-valid-styles', rule.default, {
         },
       ],
     },
+    {
+      code: `
+        import stylex from 'stylex';
+        const styles = stylex.create({
+          invalidStyle: {
+            marginStart: '10px',
+            marginEnd: '5px',
+            marginHorizontal: '15px',
+            marginVertical: '15px',
+            paddingStart: '10px',
+            paddingEnd: '5px',
+            paddingHorizontal: '5px',
+            paddingVertical: '15px',
+          },
+        });
+      `,
+      errors: [
+        {
+          message:
+            'The key "marginStart" is not allowed by stylex. Use "marginInlineStart" instead.',
```

Comment:
Maybe something like this:

"The key 'marginStart' is not a standard CSS property. Did you mean 'marginInlineStart'?"

Communicating that it's non-standard is useful context for people